modules/util/svg/svg.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
```

Comment:
How about rename `modules/util` to `modules/image` so that we can put all image related packages there.